survey-results/Readme.md

```diff
@@ -0,0 +1,318 @@
+# Rust CLI Survey
+
+Over the month of March 2018, we've been accepting responses to the Rust CLI Survey. This survey was designed to give us some areas of focus, according to the community, for the [CLI Working Group](https://github.com/rust-lang-nursery/cli-wg) (CLI-WG).
+
+One of the goals of Rust 2018 is to make writing command line applications in Rust as frictionless (and fun!) as possible. And we are super excited to say: e've received 1,045 responses! The results, while varied, paint a pretty clear picture for tangible goals.
+
+Because the CLI-WG is relatively small, unfortunately we cannot address all concerns raised in this survey. But with posts like this one, we hope to raise some awareness into what the community is collectively looking for, and things that you can use in your own crates and projects.
+
+We will also be using these results to guide our solutions when conducting any official reviews, or contributions as part of the CLI-WG.
+
+Before we dig in to the results, let's first define a CLI, [according to the working group](https://github.com/rust-lang-nursery/cli-wg/issues/4), in order to properly limit the scope.
+
+> For our intents and purposes, a CLI is any program that
+>
+> * Launches in a terminal
+> * Accepts configuration from various sources, such as command line arguments, environment variables, or configuration files
+> * Runs to completion with minimal/no user interaction
+> * Accepts input from stdin, files, or network
+> * Performs processing on some input (files, network, stdin) based on the configuration specified
+> * Communicates via standard outputs (files, network, std{out,err})
+>
+> (We specifically don't want to focus on "TUI" apps right now.)
+
+This may not be an all encompassing definition, but it's what we're using for the time being in order to limit the scope. Some (even many) survey respondents did comment on TUIs (i.e., programs that present an interactive command line interface, like `vim` or `top`), so we want to say that we acknowledge the gap, and may look to assist with this in the future. But for the time being we are calling that "out of scope" for this working group (with too large a scope we may never accomplish anything!).
+
+With that out of the way, let's walk through some of the responses, and what we're doing with the results.
+
+## Responses
+
+Starting with the demographics, it was surprising (and extremely appreciated) that around 44% of survey respondents have been using Rust for less than a year, or haven't used Rust at all! This great because it gives us "fresh eyes" to see issues that long time Rustaceans may gloss over because they've solved it long ago. However, these issues not only hurt adoption, but create needless friction that could be smoothed over.
```

Comment:
`they've solved it long ago.` -> `they've resolved (or found a work-around) long ago.`

The previous implies it's not a real issue, which in many cases isn't true.